sensor_test.py

```diff
@@ -99,13 +115,30 @@ def test_set_sensor_reading_interval(
     ), "Sensor reading did not change after waiting for specified interval"
 
 
+
+def test_set_invalid_sensor_reading_interval(get_sensor_info, set_sensor_reading_interval):
+    
+    log.info("Get original sensor reading interval")
+    original_interval = get_sensor_info().reading_interval
+    invalid_interval = -1
+    log.info("Request reading interval update to invalid interval")
+    set_sensor_reading_interval(invalid_interval)
+    log.info("Check that sensor responds with an error")
```

Comment:
This log message needs to go above line 125.

The logic for checking whether the sensor responds with an error is the same as I described in one of the comments above.